crates/bevy_sprite/src/mesh2d/material.rs

```diff
@@ -133,6 +137,12 @@ pub trait Material2d: AsBindGroup + Asset + Clone + Sized {
     }
 }
 
+#[derive(Clone, Copy, Debug, Reflect, PartialEq, Eq)]
+pub enum AlphaMode2d {
```

Comment:
Missing docs. In addition to "what is this", it would be helpful to call out what entity types this works with.